polus-precompute-volume-plugin/plugin.json

```diff
@@ -0,0 +1,78 @@
+{
+    "name": "Polus Precompute Volume Plugin",
+    "version": "0.3.0",
+    "containerId": "labshare/polus-precompute-volume-plugin:0.3.0",
+    "title": "WIPP Widget",
+    "description": "This plugin generates precomputed volumes and meshes for labelled data to view in Neuroglancer",
+    "author": "Madhuri Vihani (Madhuri.Vihani@axleinfo.com), Nicholas Schaub (Nick.Schaub@nih.gov), Hythem Sidky (Hythem.Sidky@nih.gov)",
+    "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+    "repository": "https://github.com/labshare/polus-plugins",
+    "website": "https://ncats.nih.gov/preclinical/core/informatics",
+    "citation": "https://github.com/hsidky/neurogen",
```

Comment:
This should not be pointed to a private repo.

Reply:
how do I cite Neurogen?